docs/gen-ai/non-normative/examples-llm-calls.md

```diff
@@ -715,11 +736,68 @@ sequenceDiagram
 
 <span id="gen-ai-input-messages-built-in-tools">`gen_ai.input.messages` value</span>
 
-TODO
+```json
+[
+  {
+    "role": "system",
+    "parts": [
+      {
+        "type": "text",
+        "content": "You are a helpful bot"
+      }
+    ]
+  },
+  {
+    "role": "user",
+    "parts": [
+      {
+        "type": "text",
+        "content": "Write Python code that generates a random number, executes it, and returns the result."
+      }
+    ]
+  }
+]
+```
 
 <span id="gen-ai-output-messages-built-in-tools">`gen_ai.output.messages` value</span>
 
-TODO
+```json
+[
+  {
+    "role": "assistant",
+    "parts": [
+      {
+        "type": "tool_call",
+        "id": "call_VSPygqKTWdrhaFErNvMV18Yl",
+        "name": "code_interpreter",
+        "tool_call": {
+          "type": "code_interpreter",
```

Comment:
should this be provider-specific or generic? are code_interpreter tools are similar ?